data_filter_mongodb/example.rego

```diff
@@ -0,0 +1,25 @@
+package details.authz
+
+allow {
+  input.method == "GET"
+  employee := data.employees[_]
+  employee.name == input.user
+  employee.name == input.path[1]
+}
+
+allow {
+  input.method == "GET"
+  employee := data.employees[_]
+  employee.manager == input.user
+  employee.name == input.path[1]
+}
+
+allow {
+  input.method == "GET"
+  input.path = ["employees"]
+  employee := data.employees[_]
+  input.user == "danerys"
```

Comment:
Can you remind me why we need to hard-code `danerys` ? Is it needed to show a particular scenario ?

Reply:
we wanted to have a scenario with `get/all` API call. This rego policy gives the response of all employees as the test data indicates danerys as head of all the employees. If the API request is to `/employees` and the requesting user is `danerys` it sends back the employee's details.